### ISSUE TITLE
🐛 Bug Report: Software templates - Create merge request for the existing gitlab repository

### ISSUE THREAD

**Issue by vinaykv23** [backstage/backstage#27498]
### 📜 Description

Hi Team Software templates, how to create merge request for the existing gitlab repository by modifying some files in existing gitlab repository.
Tried to create new repository its working, to modify the existing repository I am not getting any document how to achieve this

### 👍 Expected behavior

Hi Team Software templates, how to create merge request for the existing gitlab repository by modifying some files in existing gitlab repository.
Tried to create new repository its working, to modify the existing repository I am not getting any document how to achieve this

### 👎 Actual Behavior with Screenshots

should be able to create new update merge request for existing gitlab repository by modifying few fields

### 👟 Reproduction steps

Goto backstage left hand side menu 
Click on +Create
Get the list of Available Templates
here how to modify the existing project gitlab repository

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/backstage/backstage/blob/master/CODE_OF_CONDUCT.md)

### Are you willing to submit PR?

None

**Comment by benjdlambert** [comment]
Yes! There's a `gitlab` module that you can install in the scaffolder which will allow you create merge requests.

https://github.com/backstage/backstage/tree/master/plugins/scaffolder-backend-module-gitlab

Once that's installed you can hit `/create/actions` on your backstage instance to see the list of the new actions and there should be some examples for merge requests.

**Comment by vinaykv23** [comment]
Hi @benjdlambert 
Please assist me in how to achieve these steps
1) After creation of cluster backstage is not keeping track of the cluster created, in case of update user has to enter details again 
2) While creation of cluster in case of any errors in the Scalr/gitlab, user is not able to see what happening in the background - how to keep user in sync
3) In case for the same account already the cluster is created and user is trying to create cluster again how do we restrict
4) On the Backstage templates there will be maintenance activity, in case of any change in the terraform modules how to keep it in sync with backstage templates

**Comment by camilaibs** [comment]
> Hi [@benjdlambert](https://github.com/benjdlambert) Please assist me in how to achieve these steps
> 
> 1. After creation of cluster backstage is not keeping track of the cluster created, in case of update user has to enter details again
> 2. While creation of cluster in case of any errors in the Scalr/gitlab, user is not able to see what happening in the background - how to keep user in sync
> 3. In case for the same account already the cluster is created and user is trying to create cluster again how do we restrict
> 4. On the Backstage templates there will be maintenance activity, in case of any change in the terraform modules how to keep it in sync with backstage templates

Hi @vinaykv23, did you solve this?
